app/components/memorable_date_component.rb

```diff
@@ -122,6 +122,15 @@ def i18n_long_format(date)
     end
   end
 
+  def has_errors?
+    return false unless tag_options[:backend_validation]
```

Comment:
Do we need to have this new option `backend_validation`, or could we leave it out? The component is currently only used on In-Person Proofing screens, so it shouldn't have any unintended impact if that was the concern. And I think it'd be reasonable to let the errors be shown if any exist.

```suggestion
```

Reply:
Removed.